app/backend/db/schemas.py

```diff
@@ -0,0 +1,86 @@
+from enum import Enum 
+from typing import List, Dict, Any
+from datetime import date, time, datetime
+
+from pydantic import BaseModel
+
+
+class Mador(BaseModel):
+    name: str
+    manager: int = None
+
+    class Config:
+        orm_mode = True
+
+
+class User(BaseModel):
+    id: int
+    commander: int = None
+    reminder_time: time = None
+    english_name: str = None
+    mador: Mador = None
+    operators_id: List[int] = None
+
+    class Config:
+        orm_mode = True
+
+
+class Hierarchy(BaseModel):
+    """"Hierarchy for User and Commander."""
+    hierarchy: Dict[User, Any] = None
+
+
+class HierarchyList(BaseModel):
+    """Hierarchy for Reporter and Admin 
+        that can view more then 1 hierarchy.
+    """
+    hierarchys: List[Hierarchy]
+
+
+class StatusTypes(str, Enum):
+    required = "required"
+    not_required = "not_required"
+    not_important = "not_important"
+
+
+class DateDetails(BaseModel):
+    type: StatusTypes = None
```

Comment:
the date should also have the date itself in it

Reply:
Tho it's pretty useless consider whom ever use it has the date, I wrote it this way so I won't have to de seralisation, but maybe I should